napari/plugins/_npe2.py

```diff
@@ -200,11 +200,11 @@ def get_readers(path: str) -> Dict[str, str]:
     Returns
     -------
     Dict[str, str]
-        Dictionary of display_name to plugin_name
+        Dictionary of plugin_name to display name
     """
     pm = npe2.PluginManager.instance()
     return {
-        pm.get_manifest(reader.command).display_name: reader.plugin_name
+        reader.plugin_name: pm.get_manifest(reader.command).display_name
```

Comment:
Oh sweet mercy. 😂

Reply:
It was wrong before... lol. It makes more sense for us to index on `plugin_name`, as `display_name` may not be unique.